docs/src/content/docs/ru/guides/sidebar.mdx

```diff
@@ -402,24 +389,24 @@ starlight({
 starlight({
 	sidebar: [
 		{
-			label: 'Руководства',
+			label: 'Созвездия',
 			translations: {
-				'pt-BR': 'Guias',
+				'pt-BR': 'Constelações',
 			},
 			items: [
 				{
-					label: 'Компоненты',
+					label: 'Andromeda',
 					translations: {
-						'pt-BR': 'Componentes',
+						'pt-BR': 'Andrômeda',
 					},
-					link: '/guides/components/',
+					link: '/constellations/andromeda/',
 				},
 				{
-					label: 'Интернационализация (i18n)',
+					label: 'Scorpius',
```

Comment:
```suggestion
					label: 'Скорпио́н',
```

Should it be this word? (found it [here](https://ru.wikipedia.org/wiki/%D0%A1%D0%BA%D0%BE%D1%80%D0%BF%D0%B8%D0%BE%D0%BD_(%D1%81%D0%BE%D0%B7%D0%B2%D0%B5%D0%B7%D0%B4%D0%B8%D0%B5)) but not 100% sure about it)